### PR TITLE
First test - file name

### PR DESCRIPTION
**Pull request by chGoodchild**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -65,3 +65,9 @@ jobs:
                    -G "Ninja"
           ninja
           ninja test
+
+      - name: powerloader python based tests
+        shell: bash -l -eo pipefail {0}
+        run: |
+          export TEST_MAMBA_EXE=$(pwd)/build/powerloader
+          pytest powerloader/tests/
```
Comment (wolfv) [inline .github/workflows/main.yml:73]: I think the `pytest ./test` would work :)

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -70,4 +70,5 @@ jobs:
         shell: bash -l -eo pipefail {0}
         run: |
           export TEST_MAMBA_EXE=$(pwd)/build/powerloader
-          pytest powerloader/tests/
+          # pytest powerloader/tests
+          pytest powerloader/test/pytest/run_tests.py
```
Comment (wolfv) [inline .github/workflows/main.yml:74]: the workdir is already the repository. 

Just use `pytest ./test`

Comment (chGoodchild) [reply .github/workflows/main.yml]: oops, I saw that suggestion after pushing. The error code seems to have changed from four to five now that `pytest ./test` is in use.

File: .github/workflows/main.yml (hunk 1 above)
Comment (chGoodchild) [reply .github/workflows/main.yml]: oops, I saw that suggestion after pushing.

**Pull request merged**